### PR TITLE
complex list addition of adding an element in a given position

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

**Comment by drjova** [comment]
Screenshot 2022-01-07 at 11 19 39

**Comment by drjova** [comment]
@michamos ☝️ let me know if that's what curators want :)

File: src/complex-list-field/complex-list-field.component.html [hunk 1]
```diff
@@ -45,9 +45,15 @@
...
   <div [ngClass]="redLeftBorderClass">
     <!-- Elements -->
-    <div *ngFor="let item of paginatedItems; trackBy:trackByElement">
+    <div *ngFor="let item of paginatedItems; index as i; trackBy:trackByElement">
+
       <div class="complex-list-field-wrapper">
         <span *ngIf="shouldDisplayViewTemplate && values.get(item.index).keySeq().size != 0">
+          <div class="add-new-element">
+            <a href="javascript:void(0)" (click)="onAddNewElementAtPosition(i)">
+              Add new element in this position
```
Comment (michamos) [inline src/complex-list-field/complex-list-field.component.html:54]: Maybe simply `Add new element here`?

**Review by michamos — COMMENTED** [review]
Just to make sure: this will be enabled for authors and references, right?

**Comment by drjova** [comment]
Works on authors as well

Screenshot 2022-01-07 at 11 34 07

**Pull request merged**